documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -4,16 +4,10 @@
 
 ### New features
 
-* 
-
-### Changed features
-
-* 
-
-### Resolved issues
-
-* 
+* [company_name] [solution_name] now includes the maven embedded or shaded dependencies as part of the Bill of Materials (BOM) via the property --detect.maven.include.shaded.dependencies. See the [detect.maven.include.shaded.dependencies](properties/detectors/maven.md#maven-include-shaded-dependencies) property for more information.
+* [company_name] [solution_name] buildless mode now supports the exclusion of maven dependencies having "\<exclude\>" tags in the pom file and honours effects of dependency scope during Maven Dependency Resolution.
```

Comment:
Add the applicable detector names here too (Maven Project Inspector and/or Gradle Project Inspector).